x-pack/plugins/security_solution/public/timelines/components/side_panel/host_details/index.tsx

```diff
@@ -111,6 +111,7 @@ export const HostDetailsPanel: React.FC<HostDetailsProps> = React.memo(
           <ExpandableHostDetailsPageLink hostName={hostName} />
         </StyledEuiFlexButtonWrapper>
         <EuiSpacer size="m" />
+        <AssetCriticalitySelector entity={{ type: 'host', name: hostName }} />
```

Comment:
Consider extracting this into a constant outside of the component:

```js
const assetCriticalitySelectorEntity = { type: 'host', name: hostName };

export const HostDetailsPanel (...)
```

Reply:
Done.

Reply:
oh, my bad, I read memoize the component, not constant!